### PR TITLE
Release/0.11.0 take 2

### LINKED ISSUE #2058: Settings not loaded if WGPB version <= 1.4.0
**Describe the bug**
If the site is running an older version of WooCommerce Blocks, wcSettings for woocommerce-admin never get applied and the plugin fails.

**To Reproduce**

1. Install and activate WooCommerce Blocks <= 1.4.0
2. Go to any page.
3. Note the page not loading and the console error.

**Expected behavior**
Expected pages to continue loading.

**Screenshots**
Screen Shot 2019-04-15 at 5 41 15 PM

**Additional context**
Seems like the `woocommerce_components_settings` filter never gets called if an older version of wc blocks is active.  https://github.com/woocommerce/woocommerce/blob/14ed29b6f3b5c14223cb3447e963f92cbec21330/includes/blocks/class-wc-block-library.php#L42

### PR DESCRIPTION
**Pull request by psealock**
Release `0.11.0`

**Notable Change**
Minimum requirement for WooCommerce is now `3.6.0` due to changes with `wcSettings` now being applied by Woo instead of Woo-Blocks. See #2058 for more info.

Not too noticeable change: readme.txt whitespace changes. Append `?w=1` to the url to see actual changes.

Fixes https://github.com/woocommerce/woocommerce-admin/issues/2058

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -1,165 +1,200 @@
...
+- Dev: Update admin notes readme #2043
+- Dev: Add profile wizard base page #1933
+- Dev: Method to publish core ready features #1863
+- Dev: Add PHP autofix to pre-commit hook #2022
+- Dev: PHP Lint fixes #2023
+- Dev: Update Link in readme to Proper Location #2026
+- Dev: Split D3Chart utils/axis file #2000
+- Dev: Remove coupon_total from order stats table #1975
+- Fix: Transpile acorn-jsx package #2064
+- Fix: Inbox: Giving Feedback Notice #2019 👏 @Mr-AjayM
```
Comment (timmyc) [inline readme.txt]: Those emoji applause for props, amazing. 💪

File: readme.txt [hunk 2]
```diff
@@ -1,165 +1,200 @@
...
+WooCommerce Admin also allows store owners to customize a new dashboard screen with “performance indicators” that correspond in importance to their store’s operation.
+
+== Getting Started ==
+
+Once again: This plugin is under active development and, as such, we advise you to first install WooCommerce Admin in a staging/test environment. While this code is being run in production on sites, we recommend you first verify it works for you in a test environment, the same as any plugin for WooCommerce.
+
+= Minimum Requirements =
+
+* WordPress 5.0
+* WooCommerce 3.6.0 or greater
```
Comment (timmyc) [inline readme.txt]: We still have a check for WC version [here](https://github.com/woocommerce/woocommerce-admin/blob/master/woocommerce-admin.php#L115) should we update that also?

**Review by timmyc — COMMENTED** [review]
One question about the minimum WC version - should we enforce that via code?

File: readme.txt (hunk 2 above)
Comment (psealock) [reply readme.txt]: Great idea! I'll have to add it in the next release as this one has already been committed to the repo on .org

File: readme.txt [hunk 3]
```diff
@@ -1,165 +1,200 @@
...
+
+1. WooCommerce Admin Dashboard
+2. Activity Panels
+3. Analytics
+
+== Changelog ==
+
+= 0.11.0 2019-04-17
+
+- Dev: Extend report submenu items #2033
```
Comment (joshuatf) [inline readme.txt]: I like the idea of this prefix style.  Hopefully we can keep this going forward.

**Review by joshuatf — APPROVED** [review]
Nice job aggregating all these changes, Paul!  👏 

Would love to have the WC dependency check in this release that @timmyc mentioned, but if that's not possible, this LGTM.

File: readme.txt (hunk 2 above)
Comment (rrennick) [reply readme.txt]: You could do a 0.11.1 and add the requirement code.

File: readme.txt (hunk 3 above)
Comment (timmyc) [reply readme.txt]: I agree @joshuatf it aligns better with how it is done in Woo core, so it makes sense we conform to this structure.

**Comment by psealock** [comment]
>Would love to have the WC dependency check in this release that @timmyc mentioned

I'll create a new PR to get that check into `0.12.0`. Can we merge this?

**Comment by psealock** [comment]
>I'll create a new PR 

nvm, fixed in this one

**Pull request merged**